docs/data-review-request.md

```diff
@@ -0,0 +1,105 @@
+
+# Request for data collection review form
+
+**All questions are mandatory. You must receive review from a data steward peer on your responses to these questions before shipping new data collection.**
+
+1) What questions will you answer with this data?
+
+TODO
+
+2) Why does Mozilla need to answer these questions?  Are there benefits for users? Do we need this information to address product or business requirements?
+
+Connection speed is important for Firefox feature prioritization / decision making and it can provide information essential for advancing a business objective such as supporting OKRs.
+
+3) What alternative methods did you consider to answer these questions? Why were they not sufficient?
+
+TODO
```

Comment:
This is the first time something like this has been tried at Mozilla. We have proxies for bandwith (e.g. Firefox stub installer data), however, this will be unbiased, representative data of users best internet speeds and capturing within person variability.